### DISCUSSION TITLE
Enabling mouse buttons should disable HUD peek

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#19813] [Ideas] [answered]

**Comment by smoogipoo** [comment, marked as answer]
Sounds like something went wrong with your bindings at some point. Right click is not special and doesn't cause the HUD to peek in any way other than if it's bound in the keybinding settings.

I suggest you reset the "In Game" bindings.

**Reply by Doksha** [reply]
yeah, delete this, I forgot I have a mouse profile that switches M2 to ctrl
